<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyNetwork</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/net.png</normaloff>:/img/net.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Servers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Server 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="setupServer1_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="startServer1_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Server 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="setupServer2_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="startServer2_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Server 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="setupServer3_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="startServer3_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Server 4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="setupServer4_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="startServer4_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>544</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>376</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Clients</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Client 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="setupClient1_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="startClient1_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Client 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="setupClient2_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="startClient2_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Client 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="setupClient3_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="startClient3_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Client 4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="setupClient4_button">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="startClient4_button">
            <property name="text">
             <string>Turn On</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>549</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>376</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Database</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Server</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTableView" name="server_tableView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Client</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTableView" name="client_tableView"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
